conversation/src/main/java/com/ibm/watson/developer_cloud/conversation/v1/model/UpdateExample.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.conversation.v1.model;
+
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * UpdateExample.
+ */
+public class UpdateExample extends GenericModel {
```

Comment:
The problem here is that the updateExample operation already takes a `text` parameter that identifies the current example, so we can't transfer the `text` property from UpdateExample to the parameter list, since it would conflict.  The generator would make this transformation if there were no conflict, so we should rename one or the other of these in the swagger if that's the effect we want.